### ISSUE TITLE
Como eleitor(a) eu gostaria de ver um dicionário político que me informe sobre os principais conceitos políticos para que se eu tiver alguma dúvida em relação a alguma questão posso recorrer ao dicionário.

### ISSUE THREAD

**Issue by wab030** [Minhacps/votacidade#39]
O temas poderiam ser:
O que é privatização. O que é a câmara municipal etc.

**Comment by lguima** [comment]
André, no meu entendimento, isto seria um dos conteúdos que esperamos conseguir das organizações parceiras, então seria uma notícia/artigo listado na respectiva seção/página, não exatamente algo a ser tratado à parte.

@wab030

**Comment by wab030** [comment]
Isso mesmo Lucas, talvez se tiverem uma página a gente manda pra lá ou se eles construírem o conteúdo a gente publica.

**Comment by lguima** [comment]
Certo, precisamos alinhar a expectativa quanto a isso, anotei aqui para falarmos amanhã na reunião. Valeu!
